R/DataModel.R

```diff
@@ -281,6 +281,14 @@ naToZero <- function(x) {
   return(x)
 }
 
+# Aims to handle infinite values by replacing
```

Comment:
This may be too specific to PostgreSQL but I did encounter this in some situations...